### PR TITLE
format all files and add formatting check

### PR DESCRIPTION
**Pull request by ZachJW34**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ jobs:
       - checkout
       - node/install-packages:
           pkg-manager: yarn
+      - run: npm run format:check -- --all
```
Comment (BuckyMaler) [inline .circleci/config.yml:12]: 🧶

File: package.json [hunk 2]
```diff
@@ -54,7 +62,9 @@
     "eslint-config-prettier": "6.0.0",
     "eslint-plugin-cypress": "^2.10.3",
     "eslint-plugin-vue": "^6.2.2",
+    "husky": "^4.2.5",
```
Comment (BuckyMaler) [inline package.json:65]: Update `yarn.lock`.

Comment (ZachJW34) [reply package.json:65]: Caught me slippin

Comment (BuckyMaler) [reply package.json:65]: slippin

**Pull request merged**